Mastodon/Protocol/Provider/DataSourceProvider+NotificationTableViewCellDelegate.swift

```diff
@@ -514,10 +514,8 @@ extension NotificationTableViewCellDelegate where Self: DataSourceProvider & Aut
                 )
             case .account(let account, _):
                 await DataSourceFacade.coordinateToProfileScene(provider: self, account: account)
-            case .notification:
-                assertionFailure("TODO")
-            case .hashtag(_):
-                assertionFailure("TODO")
+            case .notification, .hashtag(_), .notificationBanner(_):
+                print("TODO")
```

Comment:
This feels a bit weird, I mean is this a case that's only covered to cover all enum cases or can this code really be executed?

Reply:
This case isn't supposed to be executed (like `hashtag` before), but due to `DataSourceItem` they just have to be implemented.

Reply:
I will change it to `assertionFailure` or `break`, I don't really mind 🙃